types/eth1.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
+// 	protoc-gen-go v1.31.0
```

Comment:
**Question:** Why was this required?

Reply:
There's no particular reason, I just installed the latest version. I'll downgrade my protoc-gen-go again